src/Redux/countries/cases.test.js

```diff
@@ -0,0 +1,36 @@
+import reducer, { addCountry } from './cases';
+
+test('should return the initial state', () => {
+  expect(reducer(undefined, {})).toEqual(
+    {
+      countries: [],
+      totalConfirmed: 0,
+    },
+  );
+});
```

Comment:
- [ ] Please can you kindly cross-check why this test is failing with the following error and make the necessary changes to get the test passing.
```
FAIL src/Redux/countries/cases.test.js
  ● should return the initial state

    expect(received).toEqual(expected) // deep equality

    - Expected  - 0
    + Received  + 1

      Object {
        "countries": Array [],
    +   "ready": false,
        "totalConfirmed": 0,
      }

      2 |
      3 | test('should return the initial state', () => {
    > 4 |   expect(reducer(undefined, {})).toEqual(
        |                                  ^
      5 |     {
      6 |       countries: [],
      7 |       totalConfirmed: 0,

      at Object.<anonymous> (src/Redux/countries/cases.test.js:4:34)

```